03_letters/lexual/freq.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func letters(s string) map[rune]int {
+	m := make(map[rune]int)
+	for _, r := range s {
+		m[r]++
+	}
+	return m
+}
+
+func sortLetters(m map[rune]int) []string {
+	keys := make([]rune, len(m))
+	i := 0
+	for k := range m {
```

Comment:
`for i, k : range ... ` may be a better fit here

Reply:
isn't the behaviour that when you use range over a map the resulting pair is key,value not index,value ?

Reply:
aww right, It's a map. sorry for that